docs/en/connector-v2/source/kafka.md

```diff
@@ -180,3 +181,65 @@ source {
 }
 ```
 
+### Multiple Kafka Source
+
+> Currently, multiple kafka source reads are supported using the Zeta engine, but note that you can only configure one instance of `bootstrap.servers`, and only one in the `table_list` and `topic` parameters,Currently, when you use multiple topics in `table_list`, you are still free to set parameters for each `topic`
+
+```hocon
+
+env {
+  execution.parallelism = 1
+  job.mode = "BATCH"
+}
+
+source {
+  Kafka {
+    bootstrap.servers = "kafka_e2e:9092"
+    table_list = [
+      {
+        topic = "^test-ogg-sou.*"
+        pattern = "true"
+        consumer.group = "ogg_multi_group"
+        start_mode = earliest
+        schema = {
+          fields {
+            id = "int"
+            name = "string"
+            description = "string"
+            weight = "string"
+          }
+        },
+        format = ogg_json
+      }, {
+        topic = "test-cdc_mds"
+        consumer.group = "canal_multi_group"
+        start_mode = earliest
+        schema = {
+          fields {
+            id = "int"
+            name = "string"
+            description = "string"
+            weight = "string"
+          }
+        },
+        format = canal_json
+      }
+    ]
+ # Each topic in the `table_list` shares the config
+    kafka.config = {
+      client.id = client_1
+      max.poll.records = 50000
+    }
+  }
+}
+
+sink {
+  Assert {
+    rules {
+      // The current table name is the same as the topic name in order to maintain fewer configuration parameters
+      table-names = ["^test-ogg-sou.*","test-cdc_mds"]
+    }
+  }
+}
```

Comment:
How about add demo of write data to doris? It's better to show how to use multi-table in one job.

Reply:
Good idea. I will add it later. I think this pr written into doris has little to do with the current pr，I'll replenish sink doris alone. What do you think

Reply:
I think we can replace `Assert` to `Doris` connector. Because this demo is useless for users.

Reply:
Ok, I will finish the e2e addition of this soon